### PR TITLE
Activate Algolia search insights in testing context. Includes additio…

### PR DESCRIPTION
**Pull request by mcstover**
…nal event passing from button components up through the controller and into the Algolia loan card adapter.

### CODE REVIEW

**Comment by mcstover** [comment]
@foochris @BoulderBrains This is the initial implementation of a conversion event that is passed to Algolia. Please have a look over this and callout any concerns.

I will add all environment configs and implement to the actual list loan card and filter page in this branch as well.

**Comment by mcstover** [comment]
@foochris @BoulderBrains This is ready for review.

File: src/pages/AlgoliaPOC.vue [hunk 1]
```diff
@@ -240,6 +258,18 @@ export default {
...
 		this.itemsInBasket = _map(_get(basketData, 'shop.basket.items.values'), 'id');
 
+		this.apollo.watchQuery({
+			query: itemsInBasketQuery,
+			variables: {
+				basketId: cookieStore.get('kvbskt'),
+			},
+		}).subscribe({
+			next: ({ data }) => {
+				console.log(data);
```
Comment (foochris) [inline src/pages/AlgoliaPOC.vue]: @mcstover console.log still needed?

Comment (mcstover) [reply src/pages/AlgoliaPOC.vue]: nope. will remove in a future pr

**Pull request merged**